dataframe_api_compat/polars_standard/group_by_object.py

```diff
@@ -58,7 +58,8 @@ def _to_dataframe(self, result: pl.DataFrame | pl.LazyFrame) -> DataFrame:
         )
 
     def size(self) -> DataFrame:
-        result = self._grouped.count().rename({"count": "size"})
+        method = "len" if hasattr(self._grouped, "len") else "count"
```

Comment:
not too keen on `hasattr` matching, could we either check against the polars version, or just bump up the minimum supported version?

Reply:
Rewritten